2021-09-21 17:00:16,514 INFO    MainThread:12035 [wandb_setup.py:_flush():69] setting env: {}
2021-09-21 17:00:16,514 INFO    MainThread:12035 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-21 17:00:16,514 INFO    MainThread:12035 [wandb_init.py:_log_setup():348] Logging user logs to /root/Source/Set-Tree/graphtree/wandb/run-20210921_170016-29vif4y1/logs/debug.log
2021-09-21 17:00:16,514 INFO    MainThread:12035 [wandb_init.py:_log_setup():349] Logging internal logs to /root/Source/Set-Tree/graphtree/wandb/run-20210921_170016-29vif4y1/logs/debug-internal.log
2021-09-21 17:00:16,515 INFO    MainThread:12035 [wandb_init.py:init():381] calling init triggers
2021-09-21 17:00:16,515 INFO    MainThread:12035 [wandb_init.py:init():386] wandb.init called with sweep_config: {}
config: {}
2021-09-21 17:00:16,515 INFO    MainThread:12035 [wandb_init.py:init():401] re-initializing run, found existing run on stack: 10bb5291
2021-09-21 17:00:16,516 INFO    MainThread:12035 [wandb_run.py:finish():1275] finishing run mlwell-rgb/gta/10bb5291
2021-09-21 17:00:16,557 INFO    MainThread:12035 [jupyter.py:save_history():353] not saving jupyter history
2021-09-21 17:00:16,557 INFO    MainThread:12035 [jupyter.py:save_ipynb():298] not saving jupyter notebook
2021-09-21 17:00:16,558 INFO    MainThread:12035 [wandb_init.py:_jupyter_teardown():281] cleaning up jupyter logic
2021-09-21 17:00:16,558 INFO    MainThread:12035 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 0
2021-09-21 17:00:16,558 INFO    MainThread:12035 [wandb_run.py:_restore():1547] restore
2021-09-21 17:00:17,425 INFO    MainThread:12035 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 973
  total_bytes: 973
}

2021-09-21 17:00:17,974 INFO    MainThread:12035 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 973
  total_bytes: 973
}

2021-09-21 17:00:18,297 INFO    MainThread:12035 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  uploaded_bytes: 973
  total_bytes: 1174
}

2021-09-21 17:00:18,400 INFO    MainThread:12035 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 973
  total_bytes: 5023
}

2021-09-21 17:00:18,505 INFO    MainThread:12035 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 973
  total_bytes: 5023
}

2021-09-21 17:00:18,607 INFO    MainThread:12035 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 973
  total_bytes: 5023
}

2021-09-21 17:00:18,711 INFO    MainThread:12035 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 973
  total_bytes: 5023
}

2021-09-21 17:00:18,813 INFO    MainThread:12035 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 973
  total_bytes: 5023
}

2021-09-21 17:00:18,916 INFO    MainThread:12035 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5023
  total_bytes: 5023
}

2021-09-21 17:00:19,020 INFO    MainThread:12035 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5023
  total_bytes: 5023
}

2021-09-21 17:00:19,122 INFO    MainThread:12035 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5023
  total_bytes: 5023
}

2021-09-21 17:00:19,227 INFO    MainThread:12035 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5023
  total_bytes: 5023
}

2021-09-21 17:00:19,330 INFO    MainThread:12035 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5023
  total_bytes: 5023
}

2021-09-21 17:00:19,433 INFO    MainThread:12035 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5023
  total_bytes: 5023
}

2021-09-21 17:00:19,536 INFO    MainThread:12035 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5023
  total_bytes: 5023
}

2021-09-21 17:00:19,762 INFO    MainThread:12035 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5023
  total_bytes: 5023
}

2021-09-21 17:00:20,657 INFO    MainThread:12035 [wandb_run.py:_wait_for_finish():1697] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 5023
  total_bytes: 5023
}
local_info {
}

2021-09-21 17:00:21,967 INFO    MainThread:12035 [wandb_run.py:_show_files():1947] logging synced files
2021-09-21 17:00:21,975 INFO    MainThread:12035 [wandb_init.py:init():430] starting backend
2021-09-21 17:00:21,975 INFO    MainThread:12035 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-09-21 17:00:21,980 INFO    MainThread:12035 [backend.py:ensure_launched():135] starting backend process...
2021-09-21 17:00:21,986 INFO    MainThread:12035 [backend.py:ensure_launched():139] started backend process with pid: 12147
2021-09-21 17:00:21,988 INFO    MainThread:12035 [wandb_init.py:init():435] backend started and connected
2021-09-21 17:00:22,007 INFO    MainThread:12035 [wandb_run.py:_label_probe_notebook():819] probe notebook
2021-09-21 17:00:22,007 INFO    MainThread:12035 [wandb_run.py:_label_probe_notebook():826] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-09-21 17:00:22,007 INFO    MainThread:12035 [wandb_init.py:init():494] updated telemetry
2021-09-21 17:00:22,008 INFO    MainThread:12035 [wandb_init.py:init():517] communicating current version
2021-09-21 17:00:23,109 INFO    MainThread:12035 [wandb_init.py:init():522] got version response 
2021-09-21 17:00:23,109 INFO    MainThread:12035 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-09-21 17:00:23,722 INFO    MainThread:12035 [wandb_init.py:init():557] starting run threads in backend
2021-09-21 17:00:24,853 INFO    MainThread:12035 [wandb_run.py:_console_start():1605] atexit reg
2021-09-21 17:00:24,854 INFO    MainThread:12035 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.WRAP
2021-09-21 17:00:24,854 INFO    MainThread:12035 [wandb_run.py:_redirect():1516] Wrapping output streams.
2021-09-21 17:00:24,856 INFO    MainThread:12035 [wandb_run.py:_redirect():1540] Redirects installed.
2021-09-21 17:00:24,856 INFO    MainThread:12035 [wandb_init.py:init():582] run started, returning control to user process
2021-09-21 17:02:07,111 INFO    MainThread:12035 [wandb_run.py:finish():1275] finishing run mlwell-rgb/Set-Tree-graphtree/29vif4y1
